### PR TITLE
docs: add description about debugging hours

### PR DESCRIPTION
**Pull request by fntlnz**
Added the debugging hours as proposed in today's office hours!

### CODE REVIEW

File: DEBUGGING_HOURS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Debugging hours
+
+Debugging hours play a role when someone wants the help of others debugging an issue
+they have with Flaco, either if it is an issue using it or developing against it!
+
+The other purpose of the debugging hours is to offload discussions from the office hours
+to a meeting solely dedicated to a specific issue.
+
+Debugging hours are not scheduled in advance, what happens is that the person that wants to propose a topic
+just opens an issue with `/kind debugging-hours` proposing a debugging topic and put your availability to do a call in it.
```
Comment (leodido) [inline DEBUGGING_HOURS.md]:
```suggestion
just opens an issue with `/kind debugging-hours` proposing a debugging topic and puts in his/her availability to do a call.
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ Falco office hours gives the Falco community a common time to come together to d
 
 Topics are proposed and voted in the office-hours repo by opening an issue, and the most upvoted issues have a chance to be addressed on the upcoming call. The less upvoted are kept for the preceeding call (or can be discussed if time allows on the current call). The MC for the week takes care of reaching out to the issue owner about participating in the upcoming call.
 
+Note: If you want to debug problems together, look at [DEBUG_HOURS.md](DEBUG_HOURS.md)
```
Comment (leodido) [inline README.md]:
```suggestion
Note: If you want to debug problems together, look at [DEBUGGING_HOURS.md](DEBUGGING_HOURS.md)
```

File: DEBUGGING_HOURS.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Debugging hours
+
+Debugging hours play a role when someone wants the help of others debugging an issue
+they have with Flaco, either if it is an issue using it or developing against it!
+
+The other purpose of the debugging hours is to offload discussions from the office hours
+to a meeting solely dedicated to a specific issue.
+
+Debugging hours are not scheduled in advance, what happens is that the person that wants to propose a topic
+just opens an issue with `/kind debugging-hours` proposing a debugging topic and puts in his/her availability to do a call.
```
Comment (fntlnz) [inline DEBUGGING_HOURS.md]: I don’t like the gender pronouns here @leodido

Comment (Kaizhe) [reply DEBUGGING_HOURS.md]: I would like to add that `reproducible environment is ready?` question or we want to make it mandatory to avoid errands

**Pull request merged**